CHANGELOG.md

```diff
@@ -2,13 +2,17 @@
 
 ## unreleased
 
+## 2.53.0-rc.1 / 2024-06-10
+
+* [BUGFIX] Rules: Fix Group.Equals() to take in account the new queryOffset too. Followup to #14061. #14273
```

Comment:
Not sure if mentioning method names will help users understand what is being fixed here. With that said, I'm struggling to find a better changelog entry 😬

Reply:
it will be removed anyway since it's fixing something introduced in the rc.0